### ISSUE TITLE
stop constantly saying "To hide the sidebar, use the icon in the top right, or press ⌘B."

### ISSUE THREAD

**Issue by jason-downs** [getcursor/cursor#1189]
Every single time I click on a menu item in the left sidebar when that panel is already open it has the message:

To hide the sidebar, use the icon in the top right, or press ⌘B.

This happens over and over, when it should perhaps be just something that pops up the first time you do it, or at least it should have a "don't show this message again" checkbox or something. Otherwise it just gets ridiculous after a while.

**Comment by kenbankspeng** [comment]
I fixed this by changing the hotkey. Mine now says ⌘D instead of ⌘B... Just as annoying though.

**Comment by Sunny-117** [comment]
The same issue, I hope that clicking on the file icon can also fold it like VSCode

**Comment by shailander** [comment]
What is the reason for it? Had they removed the binding through the icon?

**Comment by oneQiu** [comment]
yes, it is so SX

**Comment by Parkerhiphop** [comment]
I'm switching to [Windsurf Editor](https://codeium.com/) because I'm really tired of dealing with this issue.

More importantly, when writing markdown files like README.md, pressing ⌘B toggles bold text instead of the sidebar, which is incredibly frustrating.

**Comment by katlim-br** [comment]
In my opinion, the real solution is to keep it like VSCode does. If you click the icon, the panel closes.

If we are supposed to take Cursor as our default IDE then these tiny things are just friction non-sense. Everything should be as smooth.

(I also have a beef with Cursor reserving Cmd+K, but I can understand that a bit more)

**Comment by eatorres** [comment]
> In my opinion, the real solution is to keep it like VSCode does. If you click the icon, the panel closes.
> 
> If we are supposed to take Cursor as our default IDE then these tiny things are just friction non-sense. Everything should be as smooth.
> 
> (I also have a beef with Cursor reserving Cmd+K, but I can understand that a bit more)

Agreed, if Cursor already knows we have the intent of hiding the menu, it should just do so instead of telling you to use their way of doing so

**Comment by fishnux** [comment]
The Cursor team no longer monitors issues created here on GitHub. If someone has an account [on their forum](https://forum.cursor.com/) please open a feature request to make it hide (fold) the Primary Side Bar when clicking the folder icon.

**Comment by ronvoluted** [comment]
Very shoehorned UX.

**Comment by JackyCCK2126** [comment]
This hurts the user experience so much actually.
